### PR TITLE
Example data permission rename

### PR DESCRIPTION
**Pull request by ostcar**

### CODE REVIEW

**Comment by ostcar** [comment]
This PR also contains the changes made in  #5790. So the other pr has to be merged first.

I did not change the field "vote_delegated_$_to_id" since we probably undo it in the other PR.

File: docs/example-data.json [hunk 1]
```diff
@@ -400,29 +400,29 @@
...
-            "users.can_see_extra_data",
-            "users.can_see_name"
+            "assignment.can_manage",
+            "assignment.can_nominate_other",
+            "assignment.can_nominate_self",
+            "assignment.can_see",
+            "meeting.can_manage_projector",
+            "meeting.can_see_frontpage",
+            "meeting.can_see_projector",
+            "meeting.can_manage_tags",
```
Comment (FinnStutzenstein) [inline docs/example-data.json:417]: wrong name

File: docs/example-data.json [hunk 2]
```diff
@@ -400,29 +400,29 @@
...
-            "users.can_manage",
-            "users.can_see_extra_data",
-            "users.can_see_name"
+            "assignment.can_manage",
+            "assignment.can_nominate_other",
+            "assignment.can_nominate_self",
+            "assignment.can_see",
+            "meeting.can_manage_projector",
+            "meeting.can_see_frontpage",
+            "meeting.can_see_projector",
```
Comment (FinnStutzenstein) [inline docs/example-data.json:416]: .

File: docs/example-data.json [hunk 3]
```diff
@@ -400,29 +400,29 @@
...
-            "motions.can_manage",
-            "motions.can_see",
-            "users.can_manage",
-            "users.can_see_extra_data",
-            "users.can_see_name"
+            "assignment.can_manage",
+            "assignment.can_nominate_other",
+            "assignment.can_nominate_self",
+            "assignment.can_see",
+            "meeting.can_manage_projector",
```
Comment (FinnStutzenstein) [inline docs/example-data.json:414]: .

File: docs/example-data.json [hunk 4]
```diff
@@ -439,19 +439,19 @@
...
-            "core.can_see_frontpage",
-            "core.can_see_projector",
-            "mediafiles.can_see",
-            "motions.can_create",
-            "motions.can_see",
-            "motions.can_support",
-            "users.can_see_name"
+            "assignment.can_see",
+            "meeting.can_see_frontpage",
+            "meeting.can_see_projector",
```
Comment (FinnStutzenstein) [inline docs/example-data.json:449]: .

File: docs/example-data.json [hunk 5]
```diff
@@ -350,23 +350,23 @@
...
             "agenda.can_see_internal_items",
-            "assignments.can_see",
-            "core.can_see_frontpage",
-            "core.can_see_projector",
-            "mediafiles.can_see",
-            "motions.can_see",
-            "users.can_see_name"
+            "assignment.can_see",
+            "meeting.can_see_frontpage",
+            "meeting.can_see_projector",
```
Comment (FinnStutzenstein) [inline docs/example-data.json:366]: .

File: docs/example-data.json [hunk 6]
```diff
@@ -467,23 +467,23 @@
...
-            "motions.can_create",
-            "motions.can_manage",
-            "motions.can_see",
-            "motions.can_support",
-            "users.can_see_name"
+            "assignment.can_nominate_other",
+            "assignment.can_nominate_self",
+            "assignment.can_see",
+            "meeting.can_see_frontpage",
+            "meeting.can_see_projector",
```
Comment (FinnStutzenstein) [inline docs/example-data.json:480]: .

**Comment by FinnStutzenstein** [comment]
Please remove autoremove setting

**Pull request merged**